tracepoint_perf/src/input_file.rs

```diff
@@ -90,17 +90,16 @@ impl InputFile {
         let new_len = old_len + len;
         vec.reserve(len);
 
-        unsafe {
```

Comment:
Note: Split large `unsafe` block into two smaller blocks so that it's more-clear which parts are actually unsafe.